pkg/logql/log/fmt.go

```diff
@@ -90,6 +90,20 @@ var (
 	}
 )
 
+func addLineAndTimestampFunctions(currLine func() string, currTimestamp func() int64) map[string]interface{} {
+	functions := make(map[string]interface{}, len(functionMap)+2)
+	for k, v := range functionMap {
+		functions[k] = v
+	}
+	functions[functionLineName] = func() string {
+		return currLine()
+	}
+	functions[functionTimestampName] = func() time.Time {
```

Comment:
newb question: shouldn't you have a separate function for timestamp and another one for the unix timestamp?

Reply:
this "function" creates the timestamp internally as a `date` object which can be combined with other functions like `unixEpoch` and `toDate` which let you format however you would like.

I'm gonna have an additional PR to handle formatting the unix timestamp as millis or nanos

Reply:
got it, ty